src/components/Sections/InterestingFactsSection/index.tsx

```diff
@@ -41,38 +47,57 @@ class InterestingFactsSection extends BaseComponent<
           <Layout wrap>
             <LayoutItem width="full" lg={{ width: "6/12" }}>
               <div class="InterestingFactsSection--Content">
-                <Headline
-                  as="span"
-                  size="xl"
-                  weight="light"
-                  data-testid={"interestingfactssection-tagline"}
-                >
-                  {this.tagline}
-                </Headline>
-                <Headline
-                  as="h2"
-                  class="text-highlight leading-none"
-                  size="xxl"
-                  data-testid={"interestingfactssection-headline"}
-                >
-                  {this.headline}
-                </Headline>
-                <Paragraph
-                  size="lg"
-                  weight="light"
-                  data-testid={"interestingfactssection-text"}
-                >
-                  <RichText content={this.text} />
-                </Paragraph>
+                {this.$scopedSlots.tagline ? (
+                  this.$scopedSlots.tagline(this.tagline)
+                ) : (
+                  <Headline
+                    as="span"
+                    size="xl"
+                    weight="light"
+                    data-testid={"interestingfactssection-tagline"}
+                  >
+                    {this.tagline}
+                  </Headline>
+                )}
+                {this.$scopedSlots.headline ? (
+                  this.$scopedSlots.headline(this.headline)
+                ) : (
+                  <Headline
+                    as="h2"
+                    class="text-highlight leading-none"
+                    size="xxl"
+                    data-testid={"interestingfactssection-headline"}
+                  >
+                    {this.headline}
+                  </Headline>
+                )}
+
+                {this.$scopedSlots.text ? (
+                  this.$scopedSlots.text(this.text)
+                ) : (
+                  <Paragraph
+                    size="lg"
+                    weight="light"
+                    data-testid={"interestingfactssection-text"}
+                  >
+                    <RichText content={this.text} />
+                  </Paragraph>
+                )}
               </div>
             </LayoutItem>
             <LayoutItem width="full" lg={{ width: "6/12" }}>
               <Layout>
-                {this.counters.map(counter => (
-                  <LayoutItem width="full" class="mt-20 lg:mt-32">
-                    <Counter value={counter.value} label={counter.label} />
-                  </LayoutItem>
-                ))}
+                {this.$scopedSlots.counters
```

Comment:
Please wrap the slot content with the LayoutItem as well. Since this will give some general structure to the section.